dallinger/bots.py

```diff
@@ -91,7 +108,26 @@ def sign_off(self):
             logger.error("Error during experiment sign off.")
             return False
 
+    def complete_experiment(self, status):
+        tmp_driver = webdriver.PhantomJS()
```

Comment:
I don't understand the rationale. From the "Forced call" log item below I'd guess this is intended to clean up a user's state if things go catastrophically wrong, but I don't believe there's value in that.